docs/api/topics/apps.rst

```diff
@@ -280,6 +281,8 @@ App
     :type release_notes: string|object|null
     :param resource_uri: The canonical URI for this resource.
     :type resource_uri: string
+    :param size: Size of the app in bytes.
```

Comment:
Maybe `file_size` to make it more explicit what we're talking about? Just in case there's any ambiguity with screen size or something else.